app/uk/gov/hmrc/eusubsidycompliancefrontend/controllers/AccountController.scala

```diff
@@ -57,7 +59,15 @@ class AccountController @Inject()(
       _ <- businessEntityJourneyOpt.fold(store.put(BusinessEntityJourney.fromUndertakingOpt(retrievedUndertaking)))(Future.successful)
       _ <- if (retrievedUndertaking.isDefined) store.put(retrievedUndertaking.getOrElse(sys.error("Undertaking is Missing"))) else Future.successful(Unit)
     } yield (retrievedUndertaking, eligibilityJourney, undertakingJourney) match {
-      case (Some(undertaking), _, _) => Ok(accountPage(undertaking, !undertaking.getAllNonLeadEORIs().isEmpty))
+
+      case (Some(undertaking), _, _) =>
+
+        val lastDayToReportDate = undertaking.lastSubsidyUsageUpdt.map(_.plusDays(90))
+        val lastDayToReportString = lastDayToReportDate.map(TimeUtils.govDisplayFormat)
```

Comment:
This has been moved in my latest merge. You can also use the extension method provided by `DateFormatter.Syntax`

Reply:
yes, doing this as I type :)

Reply:
updated